packages/vanilla/src/components/horizontalrule/horizontalrule.template.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Horizontal Rule template literal
```

Comment:
When I see `horizontalrule.template.js` I'm thinking I'll get back a `<template>...</template/>`

Reply:
Not sure if we should change the name, as we're still using a way of templating but as a template literal. It's also much more flexible than the template tag. I honestly haven't seen any implementations of the template tag in production.

cc: @kennylam @annawen1 @andysherman2121 @sannminwin

Reply:
I'm not necessarily suggesting we change the name. I'm not sure what I'm suggesting really. Just communicating what I expected this to return I suppose. No big deal.

> I honestly haven't seen any implementations of the template tag in production.

I've seen it used in tandem with a templating library like Handlebars. I've also seen it as a main player in building [Web Components](https://css-tricks.com/an-introduction-to-web-components/). 🤷‍♂️

Reply:
To be fair, it's also because of the lack of support in IE11. With that said though, you can build out web components with either HTML templates or template literals, and from what I've seen the opinion on approach between one and the other is somewhat split. 

I like the flexibility of template literals, and it can be used today without much fuss. But it is a good discussion topic perhaps in our next huddle, and especially if/when we decide to start thinking about building web components.